Formula/orientdb.rb

```diff
@@ -24,12 +24,14 @@ def install
     inreplace %W[bin/orientdb.sh bin/console.sh bin/gremlin.sh],
       '"YOUR_ORIENTDB_INSTALLATION_PATH"', libexec
 
+    inreplace "#{libexec}/config/orientdb-server-log.properties", "../log", "#{var}/log"
```

Comment:
This file doesn't seem to exist yet: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2734/version=yosemite/testReport/junit/brew-test-bot/yosemite/install_orientdb/